### PR TITLE
add WDL task assign_clades_to_nodes; integrate with build_augur_tree workflow

### PR DESCRIPTION
**Pull request by tomkinsc**
add WDL task `assign_clades_to_nodes`; integrate with `build_augur_tree` workflow

### CODE REVIEW

File: pipes/WDL/tasks/tasks_nextstrain.wdl [hunk 1]
```diff
@@ -422,6 +422,43 @@ task translate_augur_tree {
...
+        docker: docker
+        memory: select_first([machine_mem_gb, 3]) + " GB"
+        cpu :   2
+        disks:  "local-disk 50 HDD"
+        dx_instance_type: "mem1_ssd1_v2_x2"
+        preemptible: 2
+    }
+    output {
+        File node_clade_data_json = "~{out_basename}_node-clade-assignments.json"
+        String augur_version      = read_string("VERSION")
```
Comment (dpark01) [inline pipes/WDL/tasks/tasks_nextstrain.wdl:458]: Doesn't look like you wrote the `VERSION` file in the above command block

File: pipes/WDL/workflows/build_augur_tree.wdl [hunk 2]
```diff
@@ -41,6 +42,10 @@ workflow build_augur_tree {
         ancestral_traits_to_infer: {
           description: "A list of metadata traits to use for ancestral node inference (see https://nextstrain-augur.readthedocs.io/en/stable/usage/cli/traits.html). Multiple traits may be specified; must correspond exactly to column headers in metadata file. Omitting these values will skip ancestral trait inference, and ancestral nodes will not have estimated values for metadata."
         }
+        clades_tsv: {
+          description: "A TSV file containing clade mutation positions in four columns: [clade  gene    site    alt]; see: https://nextstrain.org/docs/tutorials/defining-clades",
+          patterns: ["*.tsv"]
```
Comment (dpark01) [inline pipes/WDL/workflows/build_augur_tree.wdl:47]: maybe accept `*.txt` as well?

File: pipes/WDL/tasks/tasks_nextstrain.wdl (hunk 1 above)
Comment (schaluva) [reply pipes/WDL/tasks/tasks_nextstrain.wdl:458]: I don't know how long this task would take with larger datasets so would it be a good idea to parameterize preemptibles and cpu as well?

Comment (dpark01) [reply pipes/WDL/tasks/tasks_nextstrain.wdl:458]: Oh actually I was going to suggest the opposite--can we deparameterize the memory? It's never going to need any more than what we put here. I was thinking of deparameterizing a few of the other cheap steps as well (augur export, traits, etc) while upping the default values on the heavy steps (align, draft, refine) but that's for a separate PR.

Comment (schaluva) [reply pipes/WDL/tasks/tasks_nextstrain.wdl:458]: Yeah there isn't really a right/wrong - I think whatever works best for this specific task.

Comment (tomkinsc) [reply pipes/WDL/tasks/tasks_nextstrain.wdl:458]: Yeah, this step runs fine with our large SARS-CoV-2 tree in a VM with 4GB of RAM.

File: pipes/WDL/workflows/build_augur_tree.wdl (hunk 2 above)
Comment (tomkinsc) [reply pipes/WDL/workflows/build_augur_tree.wdl:47]: done!

**Pull request merged**